executor/point_get.go

```diff
@@ -157,6 +159,9 @@ func (e *PointGetExecutor) Close() error {
 	if e.runtimeStats != nil && e.snapshot != nil {
 		e.snapshot.DelOption(kv.CollectRuntimeStats)
 	}
+	if e.idxInfo != nil && e.tblInfo != nil {
+		e.ctx.StoreIndexUsage(e.dbName, e.tblInfo.Name.O, e.idxInfo.Name.O, 1)
```

Comment:
Use `e.tblInfo.Name.L` and `e.idxInfo.Name.L` instead?

Reply:
OK.